openapi.yaml

```
@@ -19227,6 +19231,63 @@ components:
           type: string
           x-stoplight:
             id: 6zpc45u4laog4
+    ClusterInstanceGpuInfo:
+      title: ClusterInstanceGpuInfo
+      x-stoplight:
+        id: dfdp50z2mr1cs
+      type: object
+      properties:
+        count:
+          type: integer
+          x-stoplight:
+            id: hbl5jxfcth747
+          description: The number of GPUs included in the instance.
+        name:
+          type: string
+          x-stoplight:
+            id: pregmt5l22afi
+          description: The model name of the GPU.
+        manufacturer:
+          type: string
+          x-stoplight:
+            id: a0e1rvah0xfms
+          description: The manufacturer of the GPUs in the instance.
+        memory_in_mib:
+          type: string
+          x-stoplight:
+            id: 20yjj7h8sjtdu
+          description: The total GPU memory available.
+    ClusterInstanceAttributes:
+      title: ClusterInstanceAttributes
+      x-stoplight:
+        id: nxbpo7v2ecz72
+      type: object
+      properties:
+        ' instance_category':
```

I think it's bug should be, no ?
```suggestion
        instance_category:
```